package.json

```diff
@@ -2,10 +2,16 @@
   "name": "plaid",
   "version": "2.2.0",
   "description": "A node.js client for the Plaid API",
-  "keywords": [
-    "plaid",
-    "plaid.com"
-  ],
+  "keywords": ["plaid", "plaid.com"],
```

Comment:
If we are going to be adding this to the `package.json`, can we add `prettier` to `devDependencies` and have it run with [`make lint`](https://github.com/plaid/plaid-node/blob/master/Makefile#L14)?

also, lets use `semi: true`

Reply:
sounds good to me, want me to add it in this PR?

Reply:
yeah, lets add it here @dmurawsky

Reply:
@ntindall, just a warning, prettier is going to change every file if you run it in the `make lint` usually when you run it over a whole code base like that you give it it's own PR. Are you sure you want to include it in this PR?

Reply:
hey @dmurawsky, thanks for pointing that out... I'm not too intimiately familiar with the `prettier` package (it seems nice though!).

how about just keeping this change as minimal as possible (e.g. adding the optional logo type and leaving the rest of `index.d.ts` unchanged). I was initially suprised by the size of the diff, this seems like it should be a one liner...

lets open a subsequent change to pretty everything and reduce this to just the change indicated in the PR description. we can release it subsequently to unblock you.

sorry for the back and forth.

Reply:
That sounds good @ntindall, yeah, I gotta get in the habit of checking my diffs before I PR. I'll do that now